### DISCUSSION TITLE
Can't access my account

### DISCUSSION THREAD

**Discussion by riccardo-casazza** [community/community#35079] [Other Feature Feedback, Questions, & Ideas]
Hey,
I'm trying to connect to github.com/riccardocasazza. 

I have the password, but it asks for a code which is sent to my mail in the company I was working for in the past. 
I don't have access to that mail any more. 

Can you please help?

Regards,
Riccardo

**Comment by As-anonymus** [comment]
The thing is there are two options for mail one is the primary(which u don't have the access to rtn) and then the secondary that could have been used to recover your account only if you would have added it / linked it with your account.

**Comment by As-anonymus** [comment]
https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket
you can raise a ticket for same
